site/content/en/docs/reference/kubectl-kueue.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Kubectl Kueue Plugin"
+date: 2024-04-30
+description: >
+  The kubectl-kueue plugin, kueuectl, allows you to create kueue resources such as localqueue.
+---
+
+## Installing From Release Binaries
+
+### 1. Download the latest release:
```

Comment:
This is just a comment. After we implement all basic commands, we can release kueuectl via Krew, and I can mention that here.